fe/fe-core/src/main/java/org/apache/doris/policy/StoragePolicy.java

```diff
@@ -177,29 +178,48 @@ public void init(final Map<String, String> props, boolean ifNotExists) throws An
             this.cooldownTtl = getSecondsByCooldownTtl(props.get(COOLDOWN_TTL));
         }
 
-        checkIsS3ResourceAndExist(this.storageResource);
+        checkResourceIsExist(this.storageResource);
         if (!addResourceReference() && !ifNotExists) {
             throw new AnalysisException("this policy has been added to s3 resource once, policy has been created.");
         }
     }
 
-    private static Resource checkIsS3ResourceAndExist(final String storageResource) throws AnalysisException {
-        // check storage_resource type is S3, current just support S3
-        Resource resource =
-                Optional.ofNullable(Env.getCurrentEnv().getResourceMgr().getResource(storageResource))
-                    .orElseThrow(() -> new AnalysisException("storage resource doesn't exist: " + storageResource));
+    private static Resource checkResourceIsExist(final String storageResource) throws AnalysisException {
+        Resource resource = Optional.ofNullable(Env.getCurrentEnv().getResourceMgr().getResource(storageResource))
+            .orElseThrow(()-> new AnalysisException("storage resource doesn't exist: " + storageResource)) ;
 
-        if (resource.getType() != Resource.ResourceType.S3) {
```

Comment:
Could you please add one description about that now hdfs resource supports cold heat separtion in the this [docs](https://doris.apache.org/zh-CN/docs/dev/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-RESOURCE?_highlight=resource)?